src/main/java/org/quiltmc/loader/impl/QuiltLoaderImpl.java

```diff
@@ -117,7 +122,14 @@ public void setGameProvider(GameProvider provider) {
 
 	private void setGameDir(Path gameDir) {
```

Comment:
This seems to imply the system property overrules a request by the game provider for the location of the game dir, typically not called in modern mc.

I'd guess you intended that the system property would overrule here.

Reply:
Not really sure what you mean there, none of this code touches the setting of the game directory itself whatsoever